Source/Core/VideoCommon/FramebufferShaderGen.cpp

```diff
@@ -51,17 +60,17 @@ static void EmitSamplerDeclarations(std::stringstream& ss, u32 start = 0, u32 en
   }
 }
 
-static void EmitSampleTexture(std::stringstream& ss, u32 n, const char* coords)
+void EmitSampleTexture(std::stringstream& ss, u32 n, const char* coords)
```

Comment:
This function appears to use `coords` in the same way as `EmitTextureLoad` which was upgraded to `std::string_view`, but this one wasn't. Is there a specific reason for that?